src/components/game/Board.jsx

```diff
@@ -1,25 +1,10 @@
-import React, { useState } from "react";
+import React from "react";
```

Comment:
You can remove this line altogether because this is no longer needed with functional components after React v17.